styles/planet-minecraft/catppuccin.user.css

```diff
@@ -128,6 +128,26 @@
       }
     }
 
+    .content-actions {
+          li {
+              background: @mantle !important;
+              border-color: @mantle !important;
+          }
+
+          a, li a, .link, .js_link, .js_link_m {
+              color: @text
+          }
+      
+          li a:hover, li .js_link:hover, li .link:hover {
+              background-color: @base !important;
+              color: @text !important;
+          }
+
+          li .material-icons {
+              color: @subtext1 !important;
+          }
```

Comment:
Nesting can be improved here, and let's stick with `surface0` for these buttons rather than `mantle` as it was before.